client/my-sites/plans-v2/details.tsx

```diff
@@ -86,14 +114,15 @@ const DetailsPage = ( { duration, productSlug, rootUrl, header }: DetailsPagePro
 					shortName
 						? translate( 'Great choice! Now select a {{name/}} option:', {
 								components: {
-									name: <> { shortName }</>,
```

Comment:
I couldn't find any reason for this extra space, so I removed it. If I'm mistaken, definitely let me know.